app/src/pages/inside/common/dashboardHeader/dashboardHeader.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { AddDashboardButton } from '../addDashboardButton';
+import { PageBreadcrumbs } from '../pageBreadcrumbs';
+import styles from './dashboardHeader.scss';
+
+const cx = classNames.bind(styles);
+
+export const DashboardHeader = ({ breadcrumbs }) => (
+  <div className={cx('dashboard-header')}>
+    <PageBreadcrumbs data={breadcrumbs} />
+    <AddDashboardButton />
+  </div>
```

Comment:
this component is unnecessary.  For example, in DashboardItemPage: 
```
<PageLayout breadcrumbs={getBreadCrumbs()}>
   <AddDashboardButton /> // absolutely  positioned, with a top property below zero.
   ...
   ...
</PageLayout>
```

Reply:
I disagree with this opinion since there can be many such absolutely positioned elements, and it will make markup less readable:
1. It's difficult to understand that this button is actually outside of the page content. These components are not a part of a page content, so it's strange to see them there. It's a title/header/ or whatever we may call it, but not a content.
2. It's difficult to maintain positions of many absolutely positioned components between each other.

Reply:
On other side that button functionality related to page content, not layout

Reply:
It's correct. But DashboardItemPage returns PageLayout which divides page into header and content. In header we keep page title or breadcrumbs, which are responsible for displaying of user location. All other info and functionality are placed into content block. All interactions with dashboards are placed in content block. So I can't see any reason to button of adding a new dashboard into header, which is not responsible for that.

Reply:
I suggest to do some refactoring of the `PageLayout` component and its' usages:
- Instead of the `title` prop, let's make a component `PageHeader` (move and refactor existing `PageTitle` to a new file)
- Also, move `PageContent` to a new file too (suggested name is `PageSection`)

So, pageLayout will have 3 exported components: `PageLayout`, `PageHeader`, `PageContent`.
And the usage will be like this:
```
<PageLayout>
  <PageHeader>
    {/* Some title, for example, in this case may be `DashboardHeader` */}
  </PageHeader>
  <PageSection>
    {/* page content */}
  </PageSection>
</PageLayout>
```

Also, speaking of `DashboardHeader` itself, is it better to keep this component of just place breadcrumbs and the button inside the `PageHeader` directly and give an opportunity to control layout (title on the left, buttons on the right)? Maybe we need some prop whick adds a class with `display: flex; justify-content: space-between;`? @ArtsiomYeliseyenka what do you think?

Reply:
Confirm solution of dividing PageLayout on a three components.
About positioning breadcrambs and controls in PageHeader: I suggest to define  space-between by default, because at the moment, there is no cases when such block positioned in other way. If in future we will face differ requirement, it will be easy to add modify-class prop.